.travis.yml

```diff
@@ -1,20 +1,35 @@
 # http://docs.travis-ci.com/user/languages/go/
 language: go
 
-go: 1.8
+go: 1.7
```

Comment:
I noticed that we only use go1.7 as we fetch go ourselves, so updated this to match.

Reply:
What is it for?

Reply:
@laceypm please explain your question, since otherwise I expect to mark you as a pammy user.